cms/envs/common.py

```diff
@@ -103,6 +103,9 @@
 
     GENERATE_PROFILE_SCORES,
 
+    # Enterprise service settings
+    ENTENTERPRISE_CATALOG_INTERNAL_ROOT_URL,
```

Comment:
Should this be `ENTERPRISE_CATALOG_INTERNAL_ROOT_URL`?

Reply:
Whoops. Yes, fixed